README.md

```diff
@@ -0,0 +1,181 @@
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <br/>
+
+  <h1><b>Mobile First</b></h1>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Portfolio: Mobile First <a name="about-project"></a>
+
+**Mobile First** is designed to cater to the needs of mobile users primarily. It also renders adaptively on other devices and display sizes.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Not Applicable</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Not Applicable</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Styling with media query**
+- **Social media icons**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [optional] you can as well remove the live demo section and add it when your project has been deployed.